commands/hello.md

```diff
@@ -21,20 +45,25 @@ RESP2). Clients that want to handshake the RESP3 mode need to call the
     6# "role" => "master"
     7# "modules" => (empty array)
 
-The `HELLO` command has a useful reply that will state a number of facts
-about the server: the exact version, the set of modules loaded, the client
-ID, the replication role and so forth. Because of that, and given that
-the `HELLO` command also works with "2" as argument, both in order to
-downgrade the protocol back to version 2, or just to get the reply from
-the server without switching the protocol, client library authors may
-consider using this command instead of the canonical `PING` when setting
-up the connection.
+In both cases, `HELLO`'s reply states several useful facts about the server,
+such as: versions, modules loaded, client ID, replication role and so forth.
+Because of that, and given that the `HELLO` command also works with "2" as an
+argument - both to downgrade the protocol back to version 2, or just to get the
```

Comment:
This sentence still seems to be mixing between value of 2 as a downgrade, and value of 2 as a call to HELLO "without switching" version